english/explore-outside/journey-to-flaming-tower/home/home.md

```diff
@@ -1 +1,3 @@
-While walking home you stumbled upon a field of strawberries. Not any ordinary strawberries, talking ones
\ No newline at end of file
+While walking home you stumbled upon a field of strawberries. Not any ordinary strawberries, talking ones.
+
+The strawberries are actually talking hurriedly about something in their midst, pointing with extended leaves at a small object. It is white, with red spots the size of silver dollars, and about the size of the A.W.E. Striker w/ Crankcase from G.I. Joe Attack on Cobra Island series.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.